addons/sale/report/ir_actions_report_templates.xml

```diff
@@ -165,17 +165,17 @@
                 </div>
             </div>
 
-            <div class="oe_structure"/>
```

Comment:
Are you sure the `oe_structure` div has no use at all ?
It seems to be linked to stuff from website/web_editor :thinking: 
(https://github.com/odoo/odoo/commit/dd139948f0049b3676b859495ba9b2d43436f9c1)

Reply:
`oe_structure` divs are indeed used by the editor as containers for custom content (e.g. website snippets, mass mailing snippets) - but within Odoo Studio's report editor that are meaningless

Reply:
The goal of oe_structure is to provide a "safe landing space" for custom blocks - but Odoo Studio allows to drop content in *any* container div, not just in oe_structure ones (otherwise you wouldn't be able to edit a lot of stuff in reports)